src/java/picard/analysis/artifacts/ContextAccumulator.java

```diff
@@ -20,27 +19,22 @@
 
     // are the PE reads expected to face the same direction?
     private final boolean expectedTandemReads;
-    private final Map<Transition, Map<String, AlignmentAccumulator>> artifactMap;
+    private final Map<String, AlignmentAccumulator[]> artifactMap;
```

Comment:
except that it doesn't...my mistake...make a note stating what it maps to accumulators...